client/components/plans/plans-compare/style.scss

```diff
@@ -138,6 +138,10 @@
 	&:nth-child(odd) {
 		background: $gray-light;
 	}
+
+	&.is-highlighted {
+		background: darken( $gray-light, 10% );
```

Comment:
Yeah, I was thinking a different color (versus different shade) would help highlight it a bit better, considering we're already alternating the shades.